### ISSUE TITLE
Request for patch to 3.30

### ISSUE THREAD

**Issue by ryphil** [sourcegraph/sourcegraph#23762]
@sourcegraph/distribution I am requesting the following commits be included in a patch release. They are already merged into `main`:
- https://github.com/sourcegraph/sourcegraph/commit/54634acdd7855a808472ab064de12bd81a85c67e
- https://github.com/sourcegraph/sourcegraph/commit/88481f35f3b7ffb0a328460b1af64e9cf97fd4d5

The intent of the questions below is to ensure we keep Sourcegraph high quality and [only create patch releases based on a strict criteria.](https://about.sourcegraph.com/handbook/engineering/releases#when-are-patch-releases-performed) If you can answer yes to many or most of these questions, we will be happy to create the patch release.

I have read [when and why we perform patch releases](https://about.sourcegraph.com/handbook/engineering/releases#when-are-patch-releases-performed) and answer the questions as follows:

> Are users/customers actively asking us for these changes and cannot wait until the next full release?

[Customer](https://docs.google.com/spreadsheets/d/1OEhzdMSlkGOaWyGKwdiAGlirsKKj9EN45Izn7kdKNTg/edit#gid=0&range=A4) is blocked on deployment and rollout to the trial users because of these issues. 

> Are the changes extremely minimal, well-tested, and low risk such that not testing as we do in a full release is OK?

Yes, more details can be found in GitHub issue [#427](https://github.com/sourcegraph/customer/issues/427) and [#428](https://github.com/sourcegraph/customer/issues/428).

> Is there some functionality completely broken that warrants redacting the prior release of Sourcegraph and advising users wait for the patch release?

No.  This only affects Bitbucket server deployments with a large number of repositories where users have access to a large number of those repos.

> This will interrupt our regular planned work and release cycle, taking one full working day of our time, and will take up all of our site admin's valuable time by asking them to upgrade or producing noise for them if they don't need to upgrade.
>
> Do you believe the changes are important enough to warrant this?

Yes.  [Customer](https://docs.google.com/spreadsheets/d/1OEhzdMSlkGOaWyGKwdiAGlirsKKj9EN45Izn7kdKNTg/edit#gid=0&range=A4) is blocked and will not make progress until we create this patch.  We would like to get sign off on the technical requirements asap.

> Patch releases are a signal we can do something better to improve the quality of Sourcegraph. Have you already scheduled a call (or created a google doc) to perform a [retrospective](https://about.sourcegraph.com/retrospectives) and identify ways we can improve?

Yes, we are constantly working with CEs to understand if fixes can wait until the next release rather than filing a patch release. In this case, we're bound by certain customer timelines that require updates ahead of normal release cycles. 

---

**For the [release captain](https://about.sourcegraph.com/handbook/engineering/releases#release-captain)** - after reviewing and approving this request:

- If there is [already an upcoming patch release](https://github.com/sourcegraph/sourcegraph/issues?q=is%3Aissue+label%3Arelease-tracking+), add the listed commits alongside a link to this issue
- If there is no upcoming patch release, create a new one:
  - Update [`dev/release/release-config.jsonc`](https://sourcegraph.com/github.com/sourcegraph/sourcegraph/-/blob/dev/release/release-config.jsonc) with the patch release in `upcomingRelease` and `releaseDate` (and open a PR to `main` to update it)
  - `yarn release tracking:issues`
  - Add the listed commits alongside a link to this issue to the generated [release tracking issue](https://github.com/sourcegraph/sourcegraph/issues?q=is%3Aissue+label%3Arelease-tracking+)

Comment and close this issue once the relevant commit(s) have been cherry-picked into the release branch.

**Comment by ggilmore** [comment]
@ryphil blocked on https://github.com/sourcegraph/sourcegraph/issues/24000

**Comment by bobheadxi** [comment]
this was done
